mercantile/__init__.py

```diff
@@ -4,14 +4,13 @@
 from collections import Sequence
 import math
 
-
 __version__ = '1.0.4'
 
 __all__ = [
     'Bbox', 'LngLat', 'LngLatBbox', 'Tile', 'bounding_tile', 'bounds',
     'children', 'feature', 'lnglat', 'parent', 'quadkey', 'quadkey_to_tile',
-    'tile', 'tiles', 'ul', 'xy_bounds']
-
+    'tile', 'tiles', 'ul', 'xy_bounds'
+]
```

Comment:
This is tangential to the new features, will you revert please?

It's not a bad change and is what black, the code formatter, would suggest, but it's best to keep PRs with new features super focused.